submissions/expero1/html-css-popup/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./style.css">
+	<title>Document</title>
+	
+</head>
+<body>
+	<header>
+		<div class='header-wrapper'>
+			<div class="logo">
+				<img src='img/favicon.png' alt='Kottans'>
+			</div>
+			<nav class="main-menu" aria-label="Main">
+				<ul>	
+					<li><a href='https://github.com/expero1/html-css-popup' target="_blank">source code</a></li>
+					<li><a href="#">Mail</a></li>
+					<li><a href="#">Images</a></li>
+				</ul>
+			</nav>
+      <div class="popup-menu-wrapper">
+        <input type="checkbox" id="popup-checkbox" class="popup-checkbox"  />
```

Comment:
Use [Prettier](https://prettier.io/) to fix code formatting

Reply:
@A-Ostrovnyy  Thanks. Should I do it right now? I just think that formatting the code right now will cause too many changes to be displayed. Will it be convenient to do a code review? Perhaps you should do it a little later? What do you think?

Reply:
Thanks. I used prettier for uniform file formatting.